src/mol-repr/structure/visual/bond-inter-unit-cylinder.ts

```diff
@@ -246,7 +253,7 @@ export function InterUnitBondCylinderImpostorVisual(materialId: number): Complex
                 state.updateColor = true;
             }
 
-            if (newStructure.interUnitBonds !== currentStructure.interUnitBonds) {
+            if (hasStructureVisibleBonds(newStructure, newProps) && newStructure.interUnitBonds !== currentStructure.interUnitBonds) {
```

Comment:
Should the 1st condition be `hasStructureVisibleBonds(newStructure, newProps) !== hasStructureVisibleBonds(currentStructure, currentProps)`?

Reply:
no, not needed